tree/ntuple/v7/src/RPageStorageFile.cxx

```diff
@@ -220,7 +220,77 @@ ROOT::Experimental::Detail::RPageSourceFile::RPageSourceFile(std::string_view nt
       *fMetrics.MakeCounter<RNTuplePlainCounter*> ("timeWallUnzip", "ns", "wall clock time spent decompressing"),
       *fMetrics.MakeCounter<RNTupleTickCounter<RNTupleAtomicCounter>*>("timeCpuRead", "ns", "CPU time spent reading"),
       *fMetrics.MakeCounter<RNTupleTickCounter<RNTuplePlainCounter>*> ("timeCpuUnzip", "ns",
-                                                                       "CPU time spent decompressing")
+                                                                       "CPU time spent decompressing"),
+      *fMetrics.MakeCounter<RNTupleCalcPerf*> ("bwRead", "MB/s", "bandwidth compressed bytes read per second",
+         fMetrics, [](const RNTupleMetrics &metrics) -> std::pair<bool, double> {
+            if (const auto szReadPayload = metrics.GetCounter("RPageSourceFile.szReadPayload")) {
+               if (const auto szReadOverhead = metrics.GetCounter("RPageSourceFile.szReadOverhead")) {
+                  if (const auto timeWallRead = metrics.GetCounter("RPageSourceFile.timeWallRead")) {
+                     if (auto walltime = timeWallRead->GetValueAsInt()) {
+                        double payload = szReadPayload->GetValueAsInt();
+                        double overhead = szReadOverhead->GetValueAsInt();
+                        // unit: bytes / nanosecond = GB/s
+                        return {true, (1000. * (payload + overhead) / walltime)};
+                     }
+                  }
+               }
+            }
+            return {false, -1.};
+         }
+      ),
+      *fMetrics.MakeCounter<RNTupleCalcPerf*> ("bwReadUnzip", "MB/s", "bandwidth uncompressed bytes read per second",
+         fMetrics, [](const RNTupleMetrics &metrics) -> std::pair<bool, double> {
+            if (const auto szUnzip = metrics.GetCounter("RPageSourceFile.szUnzip")) {
+               if (const auto timeWallRead = metrics.GetCounter("RPageSourceFile.timeWallRead")) {
+                  if (auto walltime = timeWallRead->GetValueAsInt()) {
+                     double unzip = szUnzip->GetValueAsInt();
+                     // unit: bytes / nanosecond = GB/s
+                     return {true, 1000. * unzip / walltime};
+                  }
+               }
+            }
+            return {false, -1.};
+         }
+      ),
+      *fMetrics.MakeCounter<RNTupleCalcPerf*> ("bwUnzip", "MB/s", "decompression bandwidth of compressed bytes per second",
```

Comment:
```suggestion
      *fMetrics.MakeCounter<RNTupleCalcPerf*> ("bwUnzip", "MB/s", "decompression bandwidth of uncompressed bytes per second",
```

One more tiny change to the description, since we are using the `RPageSourceFile.szUnzip` counter.  I can commit and merge it.